staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -104,6 +104,26 @@ const (
 	clusterNameKey = "kubernetes-cluster-name"
 )
 
+var (
+	// ipv6DualStack allows overriding for unit testing.  It's normally initialized from featuregates
+	ipv6DualStackEnabled bool
+)
+
+func init() {
+	func() {
+		// this allows the code ot launch without featuregates defined.  It is currently required for unit tests where the
+		// featuregates are not registered.  This is effectively coding by side effect and an explicit register should
+		// be eventually created instead.
+		defer func() {
+			if r := recover(); r != nil {
+				utilruntime.HandleError(fmt.Errorf("%v", r))
+			}
+		}()
+
+		ipv6DualStackEnabled = utilfeature.DefaultFeatureGate.Enabled(IPv6DualStack)
```

Comment:
is it still possible to change the value? I don't understand how this can work, flags won't have been parsed while init functions are running?

Reply:
right, this can't be initialized in init. could make it a member of the Cloud struct and set it in NewCloud

Reply:
OK, yeah, I asked for that exact thing in my other comment.

Reply:
where's @liggittlamp when you need him